tests/account/test_api.py

```diff
@@ -128,15 +128,15 @@ async def test_update_settings(data, status, spawn_client, resp_is, snapshot):
 
 
 async def test_get_api_keys(
-    fake2: DataFaker,
+        fake: DataFaker,
```

Comment:
There is something weird going on with the indentation here. This is not expected from Ruff.
I am guessing this could be a side-effect of a find-replace.